src/test/weatherData.test.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import store from '../redux/store';
+import WeatherData from '../component/WeatherData';
+
+describe('testing page elements', () => {
+  const testData = {
+    weather: [
+      { main: { temp: 18.46 } },
+      { name: 'Kenya' },
+    ],
+    Status: false,
+    error: null,
+  };
+
+  it('Test weather data', () => {
+    const weatherDetails = testData.weather;
+    expect(weatherDetails).toEqual(testData.weather);
+  });
+
+  it('Should return weather status', () => {
+    const weatherHd = testData.Status;
+    expect(weatherHd).toBe(testData.Status);
+  });
```

Comment:
- [ ] Congratulations, you are almost there now :clap: :clap: but I think you should check this test as other code reviewers have mentioned. This is currently testing a raw data (`object`) that you declared, and which really didn't show any behavior of your components or functions. Kindly look into the testing aspect, please. :pray: